src/pages/nft/mint.vue

```diff
@@ -286,7 +286,7 @@ export default Vue.extend({
 
       this.checkUnlock(transferTransactions);
 
-      this.transactionInfo.hash = transferTransactions.transaction!.txHash;
```

Comment:
Same here: avoid line-by-line assignment of object properties. If you still need to format or transform received object — then make a special filter or use typed utility static method

Reply:
We have too many different types of transactions for them to be stored as a single object. Also, not all data is always available in the first response. For example: if it's ethereum transaction then we firstly get the transaction hash and show it to the user, then we wait for the receipt where we can find the total fee used.